docs/guides/webpack.md

```diff
@@ -0,0 +1,37 @@
+# Using Webpack with Video.js
+
+video.js, and the playback technologies such as videojs-contrib-hls all work in a Webpack based build environment. Here are several configuration changes specific to Webpack that will get you up and running. The following guide was written and tested for videojs v15.9.1 and videojs-contrib-hls v5.4.0
```

Comment:
I'm not sure it's necessary to specify the versions used. It should work with all versions